pod.go

```diff
@@ -584,8 +595,8 @@ func (p *Pod) delete() error {
 		return err
 	}
 
-	if state.State != StateReady && state.State != StateStopped {
-		return fmt.Errorf("Pod not ready or stopped, impossible to delete")
+	if state.State != StateReady && state.State != StatePaused && state.State != StateStopped {
```

Comment:
@jodh-intel Wouldn't you simply kill the pod ? This is sort of what the pet vs cattle paradigm is about: A sick pod is killed and restarted, you don't want to manually inspect those guys and try to fix them.